Added links to resource specs in readme (#28)
kyma-incubator/terraform-provider-gardener: README.md

## Usage

> At the moment the resource type [`gardener_shoot`](shoot/schema_shoot.go) is supported corresponding to the [`shoot`](https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1alpha1/types_shoot.go) resource of gardener. The cloud provider specific configuration and a related example is listed in the table below:

Reviewer: ```suggestion
At the moment, the Terraform Provider for Gardener defines a resource type [`gardener_shoot`](shoot/schema_shoot.go) which is linked to the corresponding [`shoot`](https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1alpha1/types_shoot.go) Gardener resource. The table shows the cloud provider-specific configuration and related examples:
```

Author: That's not what I wanted to say. This terraform provider defines a resource type "shoot" which will then be transformed and applied to Gardener. On gardener side there is a corresponding resource which we are mapping to. I wanted to link the mapped gardener type. So it is not a "supported by" relation..

Reviewer: Can we maybe use the word "linked"?

|Cloud Provider|Specification|Example|
|---|---|---|
|Amazon Web Services|[schema.go](shoot/schema_aws.go)|[main.tf](examples/aws/main.tf)|
|Goocle Cloud Platform|[schema.go](shoot/schema_gcp.go)|[main.tf](examples/gcp/main.tf)|
|Azure|[schema.go](shoot/schema_azure.go)|[main.tf](examples/azure/main.tf)|

> -Perform the following steps to use the provider:

Reviewer: ```suggestion
Perform the following steps to use the provider:
```

> 1. Navigate to one of the `main.tf` files as listed above

Reviewer: ```suggestion
1. Go to one of the `main.tf` files listed above.
```

> 2. Edit the `main.tf` file to provide the missing parameters

Reviewer: ```suggestion
2. Edit the `main.tf` file to provide the missing parameters.
```
